src/main.ts

```diff
@@ -163,10 +166,23 @@ async function uploadTestPlan()
     let filepath = map.getTestFile();
     let filename = map.getFileName(filepath);
     var urlSuffix = "tests/"+testId+"/files/"+filename+"?api-version="+util.apiConstants.latestVersion;
-    if(map.getTestKind() == TestKind.URL){
-        urlSuffix = urlSuffix + ("&fileType="+FileType.URL_TEST_CONFIG);
-    }
-    urlSuffix = baseURL + urlSuffix;
+
+    let fileType = FileType.JMX_FILE;
+        if(map.getTestKind() == TestKind.URL){
+            fileType = FileType.URL_TEST_CONFIG;
+        }
+        else {
+            //TODO(harshanb): Change to TEST_SCRIPT for all non-URL once Locust is GA
```

Comment:
Do we need to complete this TODO now?

Reply:
Currently TM handles both file types. Will take it up in the locust flag cleanup PR later